### ISSUE TITLE
Invalid value for timer timeout error in OSX

### ISSUE THREAD

**Issue by elbowdonkey** [suurjaak/Skyperious#64]
I'm seeing this when loading a Skype database:

```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/wx-3.0-osx_cocoa/wx/_misc.py", line 1367, in Notify
    self.notify()
  File "/usr/local/lib/python2.7/site-packages/wx-3.0-osx_cocoa/wx/_core.py", line 16869, in Notify
    self.result = self.callable(*self.args, **self.kwargs)
  File "/Library/Python/2.7/site-packages/skyperious/skyperious.py", line 4851, in load_later_data
    wx.CallLater(0, sortfunc, self.list_chats)
  File "/usr/local/lib/python2.7/site-packages/wx-3.0-osx_cocoa/wx/_core.py", line 16802, in __init__
    self.Start()
  File "/usr/local/lib/python2.7/site-packages/wx-3.0-osx_cocoa/wx/_core.py", line 16816, in Start
    self.timer.Start(self.millis, wx.TIMER_ONE_SHOT)
  File "/usr/local/lib/python2.7/site-packages/wx-3.0-osx_cocoa/wx/_misc.py", line 1324, in Start
    return _misc_.Timer_Start(*args, **kwargs)
wx._core.PyAssertionError: C++ assertion "m_milli > 0" failed at ./src/osx/core/timer.cpp(69) in Start(): invalid value for timer timeout
```

This is on OSX 10.11.5 on what I assume is the latest skyperious available through 'pip install'.

I also updated brew today and installed wxpython stable 3.0.2.0.

**Comment by elbowdonkey** [comment]
I forgot to mention that when trying to view chat history on a loaded database the program crashes - here's a partial paste of the crash report:

```
Process:               Python [44835]
Path:                  /System/Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python
Identifier:            org.python.python
Version:               2.7.10 (2.7.10)
Build Info:            python-94000000000000~140
Code Type:             X86-64 (Native)
Parent Process:        fish [34309]
Responsible:           Terminal [14109]
User ID:               501

PlugIn Path:             /usr/local/opt/wxmac/lib/libwx_osx_cocoau_core-3.0.dylib
PlugIn Identifier:       libwx_osx_cocoau_core-3.0.dylib
PlugIn Version:          ??? (0)

Date/Time:             2017-02-22 18:52:46.558 -0800
OS Version:            Mac OS X 10.11.5 (15F34)
Report Version:        11
Anonymous UUID:       

Sleep/Wake UUID:       

Time Awake Since Boot: 76000 seconds
Time Since Wake:       16000 seconds

System Integrity Protection: disabled

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       KERN_INVALID_ADDRESS at 0x0000000000000000
```

**Comment by suurjaak** [comment]
Timer issue fixed in the latest version, 3.6.
